src/WingmanDataTrail/MetricVizPanel/WithUsageDataPreviewPanel.tsx

```diff
@@ -147,33 +154,80 @@ export class WithUsageDataPreviewPanel extends SceneObjectBase<WithUsageDataPrev
           singularUsageType={usageDetails[sortBy].singularUsageType}
           pluralUsageType={usageDetails[sortBy].pluralUsageType}
           icon={usageDetails[sortBy].icon as IconName}
+          usageDetails={usageDetails[sortBy].usageDetails}
         />
       </div>
     );
   };
 }
-
+// TODO: pass full MetricUsageDetailsObject into UsageSectionProps?
 interface UsageSectionProps {
   usageType: MetricUsageType;
   usageCount: number;
   singularUsageType: string;
   pluralUsageType: string;
   icon: IconName;
+  usageDetails: MetricUsageDetails;
 }
 
-function UsageData({ usageType, usageCount, singularUsageType, pluralUsageType, icon }: Readonly<UsageSectionProps>) {
+function UsageData({
+  usageType,
+  usageCount,
+  singularUsageType,
+  pluralUsageType,
+  icon,
+  usageDetails,
+}: Readonly<UsageSectionProps>) {
   const styles = useStyles2(getStyles);
 
+  let dashboardItems: Array<{ label: string; value: string; count: number }> = [];
+  if (usageDetails.usageType === 'dashboard-usage') {
+    const { dashboards } = usageDetails;
+    dashboardItems = Object.entries(dashboards)
+      .map(([name, dashboardInfo]) => ({
+        label: `${name.length > 25 ? name.substring(0, 22) + '...' : name} (${dashboardInfo.count})`, // truncate long dashboard names while preserving the count
+        value: `/d/${dashboardInfo.uid}`,
+        count: dashboardInfo.count,
+      }))
+      .sort((a, b) => b.count - a.count);
+  }
+
   return (
     <div className={styles.usageContainer} data-testid="usage-data-panel">
-      <Tooltip
-        content={`Metric is used in ${usageCount} ${usageCount === 1 ? singularUsageType : pluralUsageType}`}
-        placement="top"
-      >
-        <span className={styles.usageItem} data-testid={usageType}>
-          <Icon name={icon} /> {usageCount}
-        </span>
-      </Tooltip>
+      {usageDetails.usageType === 'dashboard-usage' ? (
+        <>
+          <Dropdown
+            placement="right-start"
+            overlay={
+              <Menu style={{ maxHeight: '200px', overflowY: 'auto' }}>
+                {dashboardItems.map((item) => (
+                  <Menu.Item key={item.value} label={item.label} onClick={() => window.open(item.value, '_blank')} />
+                ))}
+              </Menu>
+            }
+          >
+            <Button
+              variant="secondary"
+              size="sm"
+              tooltip={`Metric used in ${usageCount} dashboards. Click to view them.`}
+              className={`${styles.usageItem} ${styles.clickableUsageItem}`}
```

Comment:
```suggestion
              className={cx(styles.usageItem, styles.clickableUsageItem)}
```

[cx](https://emotion.sh/docs/@emotion/css#cx) is pretty useful for concatenating classnames.